build.gradle.kts

```diff
@@ -1,8 +1,8 @@
 plugins {
   id("uk.gov.justice.hmpps.gradle-spring-boot") version "3.3.4"
```

Comment:
So I tried bumping this to the next version 3.3.5 which brings in the latest versions of Spring.  However this seemed to stop Flywaydb (the DB migration tool) from working on application startup.

I will try looking at this separately.